editor/src/components/canvas/canvas-strategies/strategies/reparent-strategy-helpers.ts

```diff
@@ -187,13 +187,16 @@ export function reparentTarget(
 
 type ReparentSubjects = NewReparentSubjects | ExistingReparentSubjects
 
+// FIXME Does it ever make sense for this to refer to more than one element?
```

Comment:
I decided to allow multiple subjects because I thought maybe we can reuse this later for pasting.